CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Change Log
 **Upgrade notes:**
 
+- **Changed**: Dynamic scopes refactor [#96](https://github.com/gencat/participa/pull/96)
```

Comment:
"Dynamic scopes refactor and undo removal of Type customization."

Reply:
Done!